src/ts/helpers/parse.ts

```diff
@@ -65,6 +65,24 @@ export function formatMilliseconds(millis: number): string {
   return `${roundNumber(millis, 3)} ms`;
 }
 
+const SECONDS_PER_MINUTE = 60;
+const SECONDS_PER_HOUR = 60 * SECONDS_PER_MINUTE;
+const SECONDS_PER_DAY = 24 * SECONDS_PER_HOUR;
```

Comment:
Also, the constants are only for one function, but I pulled them out since I thought that made the function body more readable. I'm ok with putting them in as well.